test/Motor.Extensions.Hosting.Kafka_IntegrationTest/KafkaFixture.cs

```diff
@@ -1,30 +1,33 @@
 using System.Threading.Tasks;
-using TestContainers.Container.Abstractions;
-using TestContainers.Container.Abstractions.Hosting;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Containers;
 using Xunit;
 
 namespace Motor.Extensions.Hosting.Kafka_IntegrationTest;
 
 public class KafkaFixture : IAsyncLifetime
 {
-    private readonly GenericContainer _kafka;
+    private readonly KafkaTestcontainer _container;
+    private const int KafkaPort = 9093;
 
     public KafkaFixture()
     {
-        _kafka = new ContainerBuilder<KafkaContainer>()
+        _container = new TestcontainersBuilder<KafkaTestcontainer>()
+            .WithExposedPort(KafkaPort)
```

Comment:
Can we try to use a private port binding by using `WithPortBinding(KafkaPort, true)`, instead of exposing the port directly?